opensrp-giz-malawi/src/main/java/org/smartregister/giz/presenter/LoginPresenter.java

```diff
@@ -86,12 +92,27 @@ public void processViewCustomizations() {
             }
 
         } catch (Exception e) {
-            Log.d(TAG, e.getMessage());
+            Timber.e(e);
         }
     }
 
     @Override
     public boolean isServerSettingsSet() {
```

Comment:
Will discuss with @dubdabasoduba , as for the naming that loginPresenter implements `BaseLoginContract.Presenter` coming from core .... so i can add our own method and ignore that but then i guess it can used somewhere else

Reply:
Kindly add a separate one and leave that one as in. I guess that should work OK. Right?

Reply:
https://github.com/OpenSRP/opensrp-client-anc/issues/381